### ISSUE TITLE
Accessibility: Improve Color Contrast in Dark Mode (WCAG 2.1 AA Compliance)

### ISSUE THREAD

**Issue by Biswas-Samrat** [devsecopsmaturitymodel/DevSecOps-MaturityModel#449]
The Problem In Dark Mode, some text is too dark against the background. This makes it hard for people to read, especially those with low vision. It does not meet the WCAG 2.1 AA rule, which requires a contrast ratio of at least 4.5:1 for normal text.

**### Current Contrast:** Some dark mode elements have low contrast ratios (below 4.5:1).

### **Status:** Does not meet accessibility standards.

The Goal Is to Update the colors so that all text is bright enough to meet the AA level. This will make the website easier for everyone to use.

This is some example

### **Overview Page**

<img width="560" height="360" alt="Image" src="https://github.com/user-attachments/assets/8af98fe2-73b6-4ad7-b6b4-2c6545a7cba3" />

<img width="698" height="205" alt="Image" src="https://github.com/user-attachments/assets/3261d4b1-1225-4a5f-aa56-5275d4122706" />

**### Matrix Page**

<img width="1403" height="638" alt="Image" src="https://github.com/user-attachments/assets/3f5a00f9-b3b1-4f19-a31d-adf847e4e715" />

<img width="834" height="297" alt="Image" src="https://github.com/user-attachments/assets/69bbe22a-6c23-460a-9fc6-697ce4528036" />

### **Teams Page**

<img width="891" height="399" alt="Image" src="https://github.com/user-attachments/assets/7998e004-9e5b-44da-916e-5951194a9d4b" />

@wurstbrot Assign this Issue to me
